test/testdata/asasalint.go

```diff
@@ -0,0 +1,17 @@
+package testdata
```

Comment:
```suggestion
//args: -Easasalint
package testdata
```

Reply:
thanks, I was copy from the other lint file.

is this a standrad? can we add to some contribute docs?

Reply:
All the test files contain this.

Reply:
thanks a lot time to review, 

i have read  https://golangci-lint.run/contributing/new-linters/ , and ignore the fifth, and still miss many key points in how to add linter 😂 

cloud we use a pull request template for contributor do some self-check ?
- the test file must add `//args: -Eforbidigo` or etc config for test
- need add comment for where report as an Error in test file
- the `mapstructure` tag must be `-` not `_` , and should keep in order

split the hard part in your check comment for user to do some self-check, and the soft part (need human review) keep going.